osu.Game/Overlays/NowPlayingOverlay.cs

```diff
@@ -469,5 +471,98 @@ protected override void OnHoverLost(HoverLostEvent e)
                 base.OnHoverLost(e);
             }
         }
+
+        private partial class ScrollingTextContainer : CompositeDrawable
+        {
+            private const float initial_move_delay = 1000;
+            private const float pixels_per_second = 50;
+
+            private LocalisableString text;
+            private OsuSpriteText mainSpriteText = null!;
+            private OsuSpriteText fillerSpriteText = null!;
+
+            public LocalisableString Text
+            {
+                get => text;
+                set
+                {
+                    text = value;
+                    Schedule(updateText);
+                }
+            }
+
+            public FontUsage Font
+            {
+                set =>
+                    Schedule(() =>
+                    {
+                        mainSpriteText.Font = value;
+                        fillerSpriteText.Font = value;
+
+                        updateText();
+                    });
```

Comment:
- why schedule?
- nit but I find this formatting pretty bad, would prefer

```suggestion
                set => Schedule(() =>
                {
                    mainSpriteText.Font = value;
                    fillerSpriteText.Font = value;

                    updateText();
                });
```